### PR TITLE
Make some changes for newer versions of uncrustify

### PR DESCRIPTION
**Pull request by ahcorde**
Uncrustify 0.78 is the new version in Ubuntu 24.04. It has some small differences in indentation compared to Uncrustify 0.72, which is what is in Ubuntu 22.04.

CI was failing https://build.ros2.org/job/Rdev__rviz__ubuntu_noble_amd64/6/#showFailuresLink

### CODE REVIEW

**Review by mjcarroll — APPROVED** [review]

**Comment by clalancette** [comment]
Let's wait on this one until we land https://github.com/ament/ament_lint/pull/475 ; that should smooth over at least some of these differences.

**Comment by clalancette** [comment]
OK, so I ended up merging in https://github.com/ament/ament_lint/pull/475 today.  With that in place, as far as I can tell there are no uncrustify errors here.  @ahcorde Can you confirm that, and if that is the case for you, close this PR?  Thanks.

**Pull request closed without merging**
